tests/network-tests/src/Api.ts

```diff
@@ -311,6 +315,22 @@ export class Api {
     return this.api.consts.babe.expectedBlockTime
   }
 
+  public async getBabeAuthorities(): Promise<string[]> {
+    return await this.api.query.babe.authorities().then((key) => key.map(([k]) => k.toString()))
```

Comment:
```suggestion
    return (await this.api.query.babe.authorities()).map(key => key.toString())
```
.. is easier on the eyes :)